lib/ipmi_backend_utils.pm

```diff
@@ -528,4 +529,21 @@ sub set_grub_terminal_and_timeout {
     }
 }
 
+sub reconnect_when_ssh_console_broken {
+
+    # Switch to sol console to check serial console output
+    # It is useful in the case of host crash and reboot
+    record_info("WARN", "ssh connection is broken and switch to SOL console", result => 'fail');
+    select_console 'sol', await_console => 0;
+    save_screenshot;
```

Comment:
I think this ```save_screenshot``` can be removed. Because you do not wait console ```await_console => 0```, so screenshot taken is not what you desire.

Reply:
ok.